.pre-commit-config.yaml

```diff
@@ -3,23 +3,19 @@ repos:
     rev: 19.10b0
     hooks:
     -   id: black
-        language_version: python3
 -   repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.3
     hooks:
     -   id: flake8
-        language: python_venv
```

Comment:
Same, they already specify this here https://gitlab.com/pycqa/flake8/-/blob/master/.pre-commit-hooks.yaml

and according to pre-commit docs https://pre-commit.com/

> The python_venv language is now an alias to python